e2e/common/test_common.go

```diff
@@ -352,3 +352,49 @@ func TestRolloutRestart(t *testing.T, statefulSet bool) {
 	err = e.Run()
 	require.NoError(t, err, "failed to run environment")
 }
+
+func TestReplaceHelm(t *testing.T) {
+	t.Parallel()
+	testEnvConfig := GetTestEnvConfig(t)
+	cd, err := chainlink.NewDeployment(1, map[string]any{
+		"chainlink": map[string]interface{}{
+			"resources": map[string]interface{}{
+				"requests": map[string]interface{}{
+					"cpu": "1000m",
```

Comment:
Can probably lower this a good bit.

Reply:
Messed up and hit approve instead of request changes. This is a thing I'd like to change